src/Test/Cpanel/ClientActionsTest.php

```diff
@@ -37,7 +41,12 @@ public function setup()
         $this->mockPartialZoneSet = $this->getMockBuilder('CF\Cpanel\Zone\Partial')
             ->disableOriginalConstructor()
             ->getMock();
-        $this->mockCpanelIntegration = new CpanelIntegration($this->mockConfig, $this->mockCpanelAPI, $this->mockDataStore, $this->mockLogger);
+        $this->mockRequest = $this->getMockBuilder('CF\API\Request')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->mockCpanelIntegration = new DefaultIntegration($this->mockConfig, $this->mockCpanelAPI, $this->mockDataStore, $this->mockLogger);
+        $this->clientActions = new ClientActions($this->mockCpanelIntegration, $this->mockClientAPI, $this->mockRequest);
+        $this->clientActions->setPartialZoneSet($this->mockPartialZoneSet);
```

Comment:
I dont think this should be in the constructor since it's only used in a few tests.